docs/more-info/iOS.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "iOS"
+description: "Our iOS SDK allows you to receive Apple HealthKit data via webhooks"
+---
```

Comment:
Should we call this whole section Apple Health instead of iOS? I think people's tendency would be to seek out an Apple Health doc.